app/javascript/orangelight/bookmark_all.es6

```diff
@@ -22,6 +22,16 @@ export default class BookmarkAllManager {
         this.prepopulate_value();
       }
     });
+    $('.bookmark_all').on('keydown', (e) => {
+      e.preventDefault();
```

Comment:
This line causes a keyboard trap -- once a user has tabbed to the "Bookmark all" button, they cannot Tab or Shift-Tab to any other element after that!  This is a [WCAG level A violation](https://www.w3.org/WAI/WCAG21/Understanding/no-keyboard-trap.html), and I would argue that it is even more impactful than the original bug.

If we listen for the `click` event instead, we don't have to prevent any defaults or identify the pressed key.  It will provide the spacebar and enter key automatically.

Reply:
Will pressing spacebar or enter trigger the click? I prevented the default because pressing space would scroll the screen down.

Reply:
[Yes, MDN says](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event):

> An element receives a click event when any of the following occurs:[...]
> Any user interaction that is equivalent to a click, such as pressing the Space key or Enter key while the element is focused. Note that this only applies to elements with a default key event handler, and therefore, excludes other elements that have been made focusable by setting the [tabindex](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) attribute.